### ISSUE TITLE
ToolbarFlex: search field accessibility issue

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#4860]
**Describe the bug**
When focus moves into the "Search" input in the toolbar, the left and right arrow keys no longer function to move focus to other items within the toolbar. 

**To Reproduce**

1. Go to https://master-enterprise.demo.design.infor.com/components/toolbar-flex/example-favor-title.html or on your local enterprise-ng serve http://localhost:4200/ids-enterprise-ng-demo/toolbar-flex-basic
2. try to navigate to the toolbar via tab and try to go to the search field using the arrows, then you cannot move the focus to the other toolbar button from the search field with the arrow keys

**Expected behavior**
[VPAT Recommendation]
When using role="toolbar", ensure that the nested controls can be cycled using the left and right arrow keys. As this keystroke functionality might disrupt the expected keystroke functionality within the "Search" input, developers should consider a different implementation or location of the "Search" input, or remove the "toolbar" role and keyboard functionality so that all controls may receive focus with the Tab key. 

**Version**

- ids-enterprise: 4.38.0-dev

**Screenshots**
attached screen recording when tested on enterprise-ng http://localhost:4200/ids-enterprise-ng-demo/toolbar-flex-basic
https://user-images.githubusercontent.com/29673378/108476324-da855c80-72cc-11eb-886f-2dd5ac985111.mp4

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable): NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: v85.0

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34408

**Comment by tmcconechy** [comment]
What we did for the search field is to allow up and down arrow to navigate like left and right arrow when the search field is used on a toolbar.

However the search field is optional. So you can take it off on your end if you just want to remove the searchfield and place it out of the toolbar.

I cant think of any other solution keeping it where it is. So not sure we can resolve this one.

**Comment by tmcconechy** [comment]
How about an option to make the toolbar tabbable - if you have a small number of items that might be ok? Might have to review this one with the Level Access team. Can we lower priority?

**Comment by tmcconechy** [comment]
Closing this one for now due to lack of activity and not having any other options. But the toolbar now has a tabbable option so maybe you can use this.

**Comment by pwpatton** [comment]
would be nice to use left/right arrow inside the search field once the beginning/end of the text has been reached so it acts natural to the toolbar keyboard navigation

**Comment by tmcconechy** [comment]
It's using up and down arrow for left and right - because the arrow key works in the field to move. So try up/down (its apparently correct)
